tests/Unit/Adapter/ToolsTest.php

```diff
@@ -49,7 +49,11 @@ public function provideTestCasesForBcAdd(): iterable
      */
     public function testRefreshCaCertFile(): void
     {
-        @unlink(_PS_CACHE_CA_CERT_FILE_);
+        if (file_exists(dirname(_PS_CACHE_CA_CERT_FILE_))) {
```

Comment:
Shouldn't it be `file_exists(_PS_CACHE_CA_CERT_FILE_))` before "unliking" the file ?
:thinking: